paperqa/utils.py

```diff
@@ -185,7 +215,7 @@ def flatten(iteratble: list) -> list:
 
 def get_loop() -> asyncio.AbstractEventLoop:
     try:
-        loop = asyncio.get_event_loop()
+        loop = asyncio.get_running_loop()  # new in python 3.10
```

Comment:
```suggestion
        loop = asyncio.get_running_loop()
```

Feel free to ignore this PR comment, but imo this code comment is just distracting